tests/cupy_tests/sorting_tests/test_sort.py

```diff
@@ -650,6 +650,22 @@ def test_argpartition_multi_dim(self, xp, dtype):
                 a[rows, cols, idx[:, :, kth + 1:]]).all()
         return idx[:, :, kth:kth + 1]
 
+    @testing.for_all_dtypes(no_bool=True)
+    @testing.numpy_cupy_array_equal()
+    def test_argpartition_multi_dim_kernel(self, xp, dtype):
+        a = testing.shaped_random((3, 3, 256), xp, dtype, 100)
+        kth = 2
```

Comment:
I think we can take a larger number in 256.

```suggestion
        kth = 20
```